networking/external_dns_operator/nw-control-dns-records-public-hosted-zone-aws.adoc

```diff
@@ -0,0 +1,63 @@
+[id="nw-control-dns-records-public-hosted-zone-aws_{context}"]
+= Controlling DNS records on an public hosted zone for AWS by using Red Hat External DNS Operator
+
+.Procedure
+
+. Export the kubeconfig file.
++
+[source,terminal]
+----
+$ export KUBECONFIG=~/<path to>/kubeconfig
+----
+
+. Check the user. The user must have access to the `kube-system` namespace if you don’t have the credentials as you can fetch the credentials from the `kube-system` namespace to use the cloud provider client.
```

Comment:
Why do we ever need these 2 steps here?
We don't specify any credentials in the CR below, so CCO will bring the needed secret. We don't use `aws` client in this chapter.

Reply:
@xenolinux please remove these steps as i had mentioned to add them in azure and gcp and not aws.